### PR TITLE
Rename gordo-components -> gordo

### PR DESCRIPTION
**Pull request by milesgranger**
:grimacing:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,10 +47,10 @@ See our [example](./examples) notebooks for how to develop with `gordo` locally.
 ---
 
 ## Install: 
-`pip install --upgrade gordo-components`  
+`pip install --upgrade gordo`  
 
 Bleeding edge:  
-`pip install git+https://github.com/equinor/gordo-components.git`
+`pip install git+https://github.com/equinor/gordo.git`
```
Comment (flikka) [inline README.md:52]: You want to have this in now, or in a separate one, after we move it?

**Review by flikka — COMMENTED** [review]
How much testing have you done? Would be cool to try pushing to the new pypi name, new docker names, run it on the cluster etc. But I see that it is a bit challenging perhaps to change the github url/name in the same PR? Wonder what happens if we rename the project? 

Question is I suppose, is there a way to first get in the name changes, in a PR that doesn't change the github name. Test that things work, and then rename it on github and update the urls in the codebase.

**Comment by flikka** [comment]
@epa095 are you seeing this! Look how decision strong we are when you're on holiday! LOL ROFL YOLO

File: README.md (hunk 1 above)
Comment (milesgranger) [reply README.md:52]: :man_shrugging: 
I guess my mind's eye was thinking we'd make a release, merge this, and then immediately update the url and others like readthedocs/codecov ect as those are more 'GUI' changes?

**Comment by milesgranger** [comment]
> How much testing have you done? 

You're looking at it, bud. :stuck_out_tongue_winking_eye: 
but I agree; can and will update docker and all that.


> Question is I suppose, is there a way to first get in the name changes, in a PR that doesn't change the github name. Test that things work, and then rename it on github and update the urls in the codebase.

Yes, there is nothing making it such that the name of the package needs to match the name of the repo. There is nothing wrong with pushing `gordo` for example to PyPi from the repo name as it is now, and that also goes for codecov and readthedocs.. those only really care about the url. But as above, I can and should still update docker and run it through the test project and maybe some other things I'm forgetting?

File: README.md [hunk 2]
```diff
@@ -13,33 +13,31 @@
...
-    <img src="https://api.dependabot.com/badges/status?host=github&repo=equinor/gordo-components" alt="Dependabot"/>
+    <img src="https://api.dependabot.com/badges/status?host=github&repo=equinor/gordo" alt="Dependabot"/>
   </a>
 </div>
 
 ---
 
 ## About:
-Gordo-Components is part of the common ML ops provided by `gordo`
+gordo is part of the common ML ops provided by `gordo`
```
Comment (ryanjdillon) [inline README.md]: This seems a little funny now.

**Review by ryanjdillon — COMMENTED** [review]
It's a beautiful thing seeing ALL those abbreviated lines now. Más bueno! :beer: :cowboy_hat_face:  :taco:

File: README.md (hunk 2 above)
Comment (milesgranger) [reply README.md]: Agreed. :sweat_smile:

**Comment by milesgranger** [comment]
and it seems to run through the test project a-okay, minus some clients failing because of datalake issues :trollface:  But can try it again later to be extra sure?

**Review by flikka — APPROVED** [review]

**Pull request merged**